tests/store/tracking/test_file_store.py

```diff
@@ -182,11 +182,11 @@ def test_search_experiments_filter_by_attribute(self):
         experiments = self.store.search_experiments(filter_string="attribute.`name` = 'a'")
         assert [e.name for e in experiments] == ["a"]
         experiments = self.store.search_experiments(filter_string="attribute.`name` != 'a'")
-        assert [e.name for e in experiments] == ["Abc", "ab", "Default"]
+        assert {e.name for e in experiments} == {"Abc", "ab", "Default"}
```

Comment:
Is the experiment order not detereministic?

Reply:
If it's not deterministic, we can also use `sorted`:

```suggestion
        assert sorted(e.name for e in experiments) == ["Abc", "ab", "Default"]
```

Reply:
It was showing different ordering between linux and windows with the default ordering change from `experiment_id` to either `last_update_time` or `creation_time`. I think this might be resolved by merging branch-2.0 though, so I'll revert these changes and only apply a `sorted()` comparison to the lists iif the platform differences exist. 
I'll let you know when I do the branch merge and conflict resolution :)

Reply:
The difference comes from the clock precision. As described in https://stackoverflow.com/questions/1938048/high-precision-clock-in-python, Python on Windows uses +- 16 milliseconds precision. 16 milliseconds is long enough to create multiple experiments.

Reply:
Just sleeping should fix the issue.